cmd/completion.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2021 The Kubernetes Authors.
```

Comment:
Hi @GrooveCS! Can you please remove this file from the PR before we can merge this?

Reply:
Understood. I it will be done shortly.